### PR TITLE
Fix decho() behavior to show info only to admins

### PR DESCRIPTION
**Pull request by R-J**
Fighting zombies: https://github.com/vanilla/vanilla/pull/1867 :-D

### CODE REVIEW

**Comment by tburry** [comment]
Since the $Permission parameter is a bit odd, would you mind adding @param comments to decho() in this PR?

**Comment by R-J** [comment]
I've tried my best, but I'd say I'm not the best for that :-/

File: library/core/functions.general.php [hunk 1]
```diff
@@ -795,11 +795,16 @@ function consolidateArrayValuesByKey($Array, $Key, $ValueKey = '', $DefaultValue
 if (!function_exists('decho')) {
     /**
      * Echo's debug variables if user is root admin.
+     * 
+     * @param mixed $Mixed The variable to echo.
+     * @param string $Prefix The text to be used as a prefix for the output.
+     * @param bool $Permission Whether or not output is visible for everyone.
+     * @return string The variable in human-readable format.
```
Comment (hgtonight) [inline library/core/functions.general.php]: This function doesn't return anything.

**Comment by R-J** [comment]
That's true... :-/

**Pull request merged**